ide-migration/server/migration/process/handle-deployables-task.mjs

```diff
@@ -43,14 +43,13 @@ export class HandleDeployablesTask extends MigrationTask {
 
             // Get names of projects with generated synonyms and add them to deployables
             const projectsWithSynonyms = migrationService.getProjectsWithSynonyms(locals);
+            const synonymsPaths = migrationService.checkExistingSynonymTypes(locals)
             if (projectsWithSynonyms) {
                 for (const projectName of projectsWithSynonyms) {
-                    const hdbSynonymFilePath = migrationService.getSynonymFilePath(projectName);
-                    const hdbPublicSynonymFilePath = migrationService.getPublicSynonymFilePath(projectName);
                     const projectDeployables = deployables.find((x) => x.projectName === projectName).artifacts;
-
-                    projectDeployables.push(hdbSynonymFilePath);
-                    projectDeployables.push(hdbPublicSynonymFilePath);
+                    for(const sPath of synonymsPaths){
```

Comment:
Use `synonymPath` instead of `sPath`. Such shortening of variable names is not good in terms of readability

Reply:
Also, I think you could use the JS spread operator like `projectDeployables.push(...synonymsPaths)` If you do it this way, there would be no use for the `for-of` loop and it would reduce the nesting